Doberman/ControlNode.py

```diff
@@ -1,7 +1,23 @@
 import Doberman
 
 
-class ValveControlNode(Doberman.ControlNode):
+class ControlNode(Doberman.Node):
+    """
+    Another empty base class to handle different database access
+    """
+    def setup(self, **kwargs):
+        super().setup(**kwargs)
+        self._log_command = kwargs['log_command']
+        self.control_target = kwargs['control_target']
+        self.control_value = kwargs['control_value']
+
+    def set_output(self, value):
+        self.logger.debug(f'Setting output to {value}')
+        if not self.is_silent:
+            self._log_command({'name': self.control_target, 'acknowledged': 0,
+                command: f'set {self.control_value} {value}'})
```

Comment:
`'command'`

Reply:
or should this be `self._log_command(f'set {self.control_value} {value}', self.control_target, ... `?

Reply:
Correct. Forgot I'd changed that.